packages/smooth_app/lib/pages/user_preferences_dev_mode.dart

```diff
@@ -306,6 +315,37 @@ class UserPreferencesDevMode extends AbstractUserPreferences {
       setState(() {});
     }
   }
+
+  Future<void> _changeCameraPostFrameCallbackDuration() async {
+    const int minValue = MLKitScannerPageState.postFrameCallBackMinDelay;
+    final int initialValue = userPreferences.getDevModeIndex(
+          userPreferencesCameraPostFrameDuration,
+        ) ??
+        minValue;
+
+    final int? result = await showDialog<int>(
+        context: context,
+        builder: (BuildContext context) {
+          return _EditValueDialog<int>(
+            label: 'Camera post frame callback duration',
+            initialValue: initialValue,
+            converter: (String value) => int.tryParse(value) ?? 0,
+            validator: (int? newValue) =>
+                newValue != null && newValue >= minValue,
+            textAlignment: TextAlign.center,
+            keyboardType: TextInputType.number,
+          );
+        });
+
+    if (result is int && result > minValue) {
```

Comment:
It won't even get to that point if we try to assign a non interger value to a int right?

Reply:
It's pretty much like if `result != null`, by stating it's an `int` and not an `int?`